server/src/internalClusterTest/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -1039,7 +1039,10 @@ public void testHistoryRetention() throws Exception {
         assertThat(recoveryState.getTranslog().recoveredOperations(), greaterThan(0));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/105122")
+    @TestLogging(
+        reason = "https://github.com/elastic/elasticsearch/issues/105122",
+        value = "org.elasticsearch.indices.recovery:TRACE, org.elasticsearch.index.shard:TRACE, org.elasticsearch.index.engine:TRACE"
```

Comment:
I think the spaces will prevent this working:

```suggestion
        value = "org.elasticsearch.indices.recovery:TRACE,org.elasticsearch.index.shard:TRACE,org.elasticsearch.index.engine:TRACE"
```